basalt/utils/api.py

```
@@ -1,6 +1,7 @@
 from typing import Dict, TypeVar, Optional, Tuple
 
 from .protocols import IEndpoint, INetworker, ILogger
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)